magento/magento2
# Fix #9924, prefill prefix and suffix in checkout shipping address

### Description

Prefill prefix and suffix in the checkout if they are present on a logged in customer.

### Fixed Issues (if relevant)

1. magento/magento2#9924 : Prefix and suffix are not prefilled in the quote shipping address

### Manual testing scenarios

1. start an order, using an existing account with a prefix and a suffix
2. visit the shipping address step in the checkout
3. prefix and suffix should be prefilled in the shipping address form for customers without an address
4. prefix and suffix should be prefilled in the NEW shipping address form for customers with addresses

### Contribution checklist
